wgsl/index.bs

```diff
@@ -7147,7 +7147,7 @@ Cube faces are square, so the x and y components of the result are equal.
 Reads a single texel from a texture without sampling or filtering.
 
 ```rust
-textureLoad(t: texture_1d<T>, coords: i32, level: i32) -> vec4<T>
+textureLoad(t: texture_1d<T>, coords: i32) -> vec4<T>
```

Comment:
Please update the description of the 'level' parameter below.

I suggest adding:

> When not specified, the texel is read from mip level 0.